src/UIComponents/SelectDataset.jsx

```diff
@@ -115,6 +115,7 @@ class SelectDataset extends Component {
                     style={styles.selectDatasetImage}
                     draggable={false}
                     className="ailab-image-hover"
+                    alt={dataset.name}
```

Comment:
I'm not sure the right practice, but is it possible we should add some context as well, like "Select [dataset.name]" or "Dataset related to [dataset.name]", etc.?

Reply:
I read a few articles and docs. There doesn't seem to be a clear definitive answer. In some cases it's considered redundant to repeat instructions for similar elements as long as the context can be inferred from other surrounding elements. In other cases, the recommendation is that more description and detail in an alt tag is preferred. 🤷‍♀️